### ISSUE TITLE
Device Location

### ISSUE THREAD

**Issue by barnesjt** [Barnesjt/appstudio-weather#2]
I'll need a way to get the device's location.

**Comment by barnesjt** [comment]
https://doc.qt.io/qt-5/qml-qtpositioning-positionsource.html

Some testing code from my explorations yesterday:
```
    PositionSource {
      id: src
      active: true
      updateInterval: 6000
      property real longi: 0.0
      property real latit: 0.0
      onPositionChanged: {
        var coord = src.position.coordinate;
        longi = coord.longitude;
        latit = coord.latitude;
      }
    }
```
This defines a PositionSource object, that updates 2 member properties every 6 seconds.

It is a bit unclear to me how often the location for the application should update. I might just have it update once on startup in this app.

**Comment by barnesjt** [comment]
Implemented in PR #11
